### PR TITLE
Update epic control copy to new control

### PR DESCRIPTION
**Pull request by jranks123**
## What does this change?
We no longer wish to have the line "and far fewer people are paying than ever". This PR removes that line and replaces it with a new copy. 

Old line :  More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast.

New Line :  More people are reading the Guardian than ever but advertising revenues across the media are falling fast. 

## Tested in CODE?

@guardian/contributions

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-pre-election.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ define([
                         contributionUrl: variant.options.contributeURL,
                         componentName: variant.options.componentName,
                         quoteSvg: quoteSvg.markup,
-                        p1: '&hellip; we have a small favour to ask. More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And <span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>. So you can see why we need to ask for your help, especially during this UK election.',
+                        p1: '&hellip; we have a small favour to ask. More people are reading the Guardian than ever but advertising revenues across the media are falling fast. And <span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>. So you can see why we need to ask for your help, especially during this UK election.',
```
Comment (desbo) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-pre-election.js]: not stricly PR feedback, but would it be better to say "More people than ever are reading the Guardian" rather than "More people are reading the Guardian than ever" ?

Comment (jranks123) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-pre-election.js]: That is a brilliant idea. I love it. If you want, you can raise it with the powers that be and create your own PR! Thanks again!

Comment (desbo) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-pre-election.js]: thank you, let's add it to the backlog and someone can pick it up for a future PR. @Mullefa seemed very interested.

**Review by desbo — APPROVED** [review]
